test/integration/users.test.js

```diff
@@ -351,6 +350,24 @@ describe("Users", function () {
   });
 
   describe("PUT /users/self/intro", function () {
+    it("should return 409 if the data already present", function (done) {
+      chai
+        .request(app)
+        .put(`/users/self/intro`)
+        .set("Cookie", `${cookieName}=${jwt}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          if (joinData()[1].length === 1) {
+            expect(res).to.have.status(409);
+            expect(res.body).to.be.a("object");
+            expect(res.body.message).to.equal("User data is already present!");
+          }
+          return done();
+        });
+    });
+
```

Comment:
Where are you inserting the data? test could be adding data to db before this and then trying to add it again in describe

Reply:
Data insertion to db is in the other test case.
Should I add this test in the same test and add only the if condition?

Reply:
lets discuss it over a call

also please try to take 2 reviews, not just mine.

Reply:
Okay👍🏻